src/AppInstallerCLICore/Command.cpp

```diff
@@ -405,14 +410,10 @@ namespace AppInstaller::CLI
     const CLI::Argument* ParseArgumentsStateMachine::NextPositional()
     {
         // Find the next appropriate positional arg if the current itr isn't one or has hit its limit.
-        if (m_positionalSearchItr != m_arguments.end() &&
+        while (m_positionalSearchItr != m_arguments.end() &&

```

Comment:
This seems like behavior changing, is it intentional? I do not see how multiquery would need to change this behavior.

Previous:
If itr is not positional or it's positional reaching limit, find next positional
Now:
If itr is not positional or it's positional reaching limit, find next positional not reaching limit

Reply:
Yes, this is intentional. As it was before we would always move to the next argument, even if it had not reached its limit, effectively making the limit 1 for all